examples/client.rs

```diff
@@ -2,7 +2,7 @@ use std::env;
 
 use http_body_util::Empty;
 use hyper::Request;
-use hyper_util::client::{connect::HttpConnector, legacy::Client};
+use hyper_util::client::legacy::{client::Client, connect::HttpConnector};
```

Comment:
To mirror what was "legacy" (0.14), I think we could keep the import to `client::legacy::{Client, connect::HttpConnector}`.

Reply:
Thanks for the review! Fixed it right now.